microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -220,6 +230,15 @@ def enable_uio_hv_generic_for_nic(node: Node, nic: NicInfo) -> None:
     echo = node.tools[Echo]
     lsmod = node.tools[Lsmod]
     modprobe = node.tools[Modprobe]
+    kernelcfg = node.tools[KernelConfig]
+
+    # check if kernel config for Hyper-V VMBus is enabled
+    config = "CONFIG_UIO_HV_GENERIC"
```

Comment:
[nit] This should be a class level constant

Reply:
this function looks like a util function, which class? Meanwhile, following existing coding style, a few lines above the `hv_uio_generic_uuid` could've been defined as a constant but is a literal variable inside the function, which is totally fine in my opinion.